website/screens/components/accordion/code/AccordionCodePage.tsx

```diff
@@ -14,10 +16,22 @@ import icons from "./examples/icons";
 import controlledAccordionGroup from "./examples/controlledAccordionGroup";
 import uncontrolledAccordionGroup from "./examples/uncontrolledAccordionGroup";
 import HeaderDescriptionCell from "@/common/HeaderDescriptionCell";
+import StatusTag from "@/common/StatusTag";
+import Link from "next/link";
 
 const sections = [
   {
     title: "Props",
+    content: (
+      <DxcAlert type="warning" size="fillParent">
+        The <Code>padding</Code> prop has been deprecated. Consider using layout
+        components like the{" "}
+        <Link href="/components/inset/" passHref>
+          <DxcLink>inset</DxcLink>
```

Comment:
Use uppercase for the first letter of the Inset, since it is a component and we usually do it.